enzy_htp/structure/structure_region/api.py

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
This is correct for this PR? It seems from the desc of the PR StructureRegion should be kinda implemtated. 

I'm ok with just a framework but just to make sure.

Reply:
It is correct. Will reorganize at a future point.

Reply:
Cool!